src/cli/dev.ts

```diff
@@ -65,7 +65,7 @@ export default function (program: RootCmd) {
             // wait for capabilities to be loaded and test names
             validateCapabilityNames(webhook.capabilities);
           } catch (e) {
-            console.error(e.message);
+            console.error(`Error validating capability names: ${e.message}`);
```

Comment:
```suggestion
            console.error(e, `Error validating capability names`);
```

Reply:
All messages updated to display complete `e`